digitalpublicgoods/synthea.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "Synthea",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://github.com/synthetichealth/synthea#license"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://github.com/synthetichealth/synthea/wiki"
```

Comment:
Impressive use and organization of the default GitHub wikis. This is a good one to refer to as an example, e.g. for Venture Fund companies aspiring to become nominees.